addons/account/views/account_dashboard_setup_bar.xml

```diff
@@ -1,12 +0,0 @@
-<odoo>
-    <record id="account_journal_dashboard_kanban_view_js" model="ir.ui.view">
```

Comment:
Forgot to say I think you'll need to add a fix upgrade script to remove this one. :slightly_smiling_face:

Reply:
nope, not required.
the file was not in the manifest (removed in 12.0 :D ), and also, it's an inherited view...

Reply:
https://github.com/odoo/upgrade/pull/6145#issuecomment-2161186036